src/chess/board.h

```diff
@@ -101,6 +101,8 @@ class ChessBoard {
   bool IsSameMove(Move move1, Move move2) const;
   // Returns the same move but with castling encoded in legacy way.
   Move GetLegacyMove(Move move) const;
+  // Returns the same move but with castling encoded in modern 😎 way.
```

Comment:
Maybe remove the smiley? Since it is outside the Unicode BMP, I can see it causing issues with some tools.